src/api/apollo/client.ts

```diff
@@ -7,15 +7,15 @@ import localResolvers from "./resolvers";
 
 import { ResultType } from "./generated/graphql";
 
-const api = process.env.GRAPHQL_API;
+const api = process.env.PLAYGROUND_API
```

Comment:
Can you please also open a PR to update the deployments:

- https://github.com/dapperlabs/flow-playground/blob/25e0593b6ee4c012aa3b2e4642bb7ef92cb05d10/Makefile#L20
- https://github.com/dapperlabs/flow-playground/blob/25e0593b6ee4c012aa3b2e4642bb7ef92cb05d10/Makefile#L34

Reply:
Done :)
https://github.com/dapperlabs/flow-playground/pull/121